### DISCUSSION TITLE
Point Blank (v129) with bookmarklets?

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1209] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
The securly link (https://tinyurl.com/goofcurly) page does not work anymore. It does not open a new page for executing the bookmarklet. Instead, it just shows this (see image below).
![image](https://github.com/user-attachments/assets/f5ecf278-bd64-4bc3-8ec4-c443da5c0b07)

**Comment by S-PScripts** [comment]
closed as these discussions are dead and you're in the relocate server
